src/imgtools/cli/query.py

```
@@ -0,0 +1,92 @@
+    default=False,
+    help="Force overwrite existing files.",
+)
+@click.help_option("--help", "-h")
+def query(  path: Path,
+            query_string: str,
+            group_by_root: bool,
+            n_jobs: int,
+            force: bool,
+            ) -> None:
+
+    from imgtools.dicom.interlacer import Interlacer, ModalityHighlighter, print_interlacer_tree
```

Ruff (0.11.9)

`imgtools.dicom.interlacer.ModalityHighlighter` imported but unused

Remove unused import: `imgtools.dicom.interlacer.ModalityHighlighter`

(F401)